### DISCUSSION TITLE
See more data in graph

### DISCUSSION THREAD

**Discussion by markuswesterberg** [RomRider/apexcharts-card#674] [Q&A]
Hi,

I have below code for my card:

type: custom:apexcharts-card
yaxis:
  - id: value
    min: ~0
    max: 10
    apex_config:
      tickAmount: 4
      decimalsInFloat: 2
  - id: price
    opposite: true
    min: ~0
    max: 400
    apex_config:
      tickAmount: 4
      decimalsInFloat: 2
header:
  show: true
  title: Elpris och konsumption
  show_states: true
  colorize_states: false
graph_span: 48h
span:
  start: day
  offset: '-1day'
now:
  show: true
  label: Nu
experimental:
  color_threshold: true
series:
  - entity: sensor.nordpool_kwh_se3_sek_0_10_0
    name: Elpris just nu
    yaxis_id: price
    type: column
    color: grey
    color_threshold:
      - value: -100
        color: '#0000ff'
      - value: -93
        color: '#0015ff'
      - value: -86
        color: '#002aff'
      - value: -79
        color: '#0040ff'
      - value: -72
        color: '#0055ff'
      - value: -65
        color: '#006aff'
      - value: -58
        color: '#0080ff'
      - value: -51
        color: '#0095ff'
      - value: -44
        color: '#00aaff'
      - value: -37
        color: '#00c0ff'
      - value: -30
        color: '#00d5ff'
      - value: -23
        color: '#00eaff'
      - value: -16
        color: '#00ffff'
      - value: -9
        color: '#00ffea'
      - value: -2
        color: '#00ffd5'
      - value: 5
        color: '#00ffc0'
      - value: 12
        color: '#00ffaa'
      - value: 19
        color: '#00ff95'
      - value: 26
        color: '#00ff80'
      - value: 29
        color: '#00ff6a'
      - value: 32
        color: '#00ff55'
      - value: 35
        color: '#00ff40'
      - value: 38
        color: '#00ff2a'
      - value: 41
        color: '#00ff15'
      - value: 44
        color: '#00ff00'
      - value: 47
        color: '#15ff00'
      - value: 50
        color: '#2aff00'
      - value: 53
        color: '#40ff00'
      - value: 56
        color: '#55ff00'
      - value: 59
        color: '#6aff00'
      - value: 62
        color: '#80ff00'
      - value: 65
        color: '#95ff00'
      - value: 68
        color: '#aaff00'
      - value: 71
        color: '#c0ff00'
      - value: 74
        color: '#d5ff00'
      - value: 77
        color: '#eaff00'
      - value: 80
        color: '#ffff00'
      - value: 83
        color: '#ffff00'
      - value: 86
        color: '#ffff00'
      - value: 95
        color: '#ffea00'
      - value: 100
        color: '#ffd500'
      - value: 105
        color: '#ffc000'
      - value: 110
        color: '#ffaa00'
      - value: 115
        color: '#ff9500'
      - value: 120
        color: '#ff8000'
      - value: 130
        color: '#ff6a00'
      - value: 140
        color: '#ff5500'
      - value: 150
        color: '#ff4000'
      - value: 160
        color: '#ff2a00'
      - value: 170
        color: '#ff1500'
      - value: 180
        color: '#ff0000'
      - value: 190
        color: '#f80000'
      - value: 200
        color: '#eb0000'
      - value: 210
        color: '#e00000'
      - value: 220
        color: '#d60000'
      - value: 230
        color: '#cc0000'
      - value: 240
        color: '#c10000'
      - value: 250
        color: '#b60000'
      - value: 260
        color: '#ac0000'
      - value: 270
        color: '#a10000'
      - value: 280
        color: '#990000'
      - value: 290
        color: '#8e0000'
      - value: 300
        color: '#830000'
      - value: 310
        color: '#800000'
    opacity: 0.8
    stroke_width: 7
    float_precision: 3
    group_by:
      func: max
      duration: 1h
      fill: last
  - entity: sensor.nordpool_kwh_se3_sek_0_10_0_average
    name: Medelspotpris
    yaxis_id: price
    type: line
    curve: stepline
    color: blue
    stroke_width: 1.5
    unit: öre/kWh
    float_precision: 2
  - entity: sensor.importerade_kwh_h
    yaxis_id: value
    type: line
    color: grey
    color_threshold:
      - value: 0
        color: red
    opacity: 1
    stroke_width: 1.7
    unit: kWh
    name: Förbrukning denna timme
    float_precision: 2
    group_by:
      duration: 1h
      func: max
      fill: zero

This displays as below:

![Översikt_–_Home_Assistant](https://github.com/RomRider/apexcharts-card/assets/144487855/e4b8126e-2621-41ec-b947-bc18b51e7eaf)

How can I adjust the code so that:

1. I see the prices for electricity also after current hour. (sensor.nordpool_kwh_se3_sek_0_10_0)
2. There is no straight red line (prediction) for consumption after current hour. (sensor.importerade_kwh_h)
3. There is no binding line between yesterday's and todays average electricity price? (sensor.nordpool_kwh_se3_sek_0_10_0_average)

**Comment by Dinez666** [comment]
Hi. 

Is it possible for you to share the yaml formating. I suck at formating and cant seem to get it correct :P 
Thanks in advance!
